libs/application/api/files/README.md

```diff
@@ -0,0 +1,7 @@
+# Application API Files
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test application-api-files` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
It would be ideal to add more documentation on this module, how it is used and the config required.